app/controllers/hubstats/deploys_controller.rb

```diff
@@ -27,6 +28,41 @@ def index
     end
 
     def show
+      @deploy = Hubstats::Deploy.find(params[:id])
+      @repo = @deploy.repo
+      @pull_requests = @deploy.pull_requests
+      @pull_request_count = @pull_requests.length
```

Comment:
These 3 instance variables should be locally scoped.  They are put in the stats instance variable to make it available to your view.  In general, you shouldn't make extra instance variables in controller actions as they cause more memory churn on each request.